source/common/json/json_streamer.h

```diff
@@ -243,7 +271,7 @@ class Streamer {
       ASSERT_THIS_IS_TOP_LEVEL;
       ASSERT(!expecting_value_);
       nextField();
-      this->streamer_.addSanitized(R"(")", key, R"(":)");
+      this->streamer_.addSanitized(Constants::DoubleQuote, key, R"(":)");
```

Comment:
I still don't understand why you don't have a named const for Colon.

Reply:
Oh, you mean colon. Because I don't know how to name the `":`. If we create an independent `Colon`, we way need to split the one addSanitized one to two calls:

```
addSanitized(
```

Reply:
Oh yeah, I guess you'll need Constants::ColonDoubleQuote so it's just one call.

This is why I think the code is more readable before this change, honestly, but I'm OK if you want to do that.

Reply:
`ColonDoubleQuote` is weird. Okay, I am convinced by you on this point. I will change all delimiters to use previous literal string.